ReSharper.FSharp/src/FSharp.Psi.Services/src/Daemon/Highlightings/FcsErrors.xml

```diff
@@ -525,5 +525,18 @@
     <Behavour overlapResolvePolicy="NONE"/>
     <QuickFix>ReplaceReturnTypeFix</QuickFix>
   </Error>
-  
+
+  <Error staticGroup="FSharpErrors" name="TypeMisMatchTuplesHaveDifferingLengths" ID="FS0001: Type mismatch. Expecting">
```

Comment:
Maybe we probably should check this inside the quick fix's `IsAvailable` method instead of creating different error types. I'll need to think about it.

Parsing the error also seem a too ad-hoc and a bit dangerous, it would be much better to work with FCS's `FSharpType` instead. It's not always possible to get that view of the expected type, though.